src/app/pages/app-list/app-list.component.html

```
@@ -1,3 +1,7 @@
+
+
+
+<app-onboarding-tour></app-onboarding-tour>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)